va_explorer/users/models.py

```diff
@@ -84,3 +89,15 @@ def save(self, *args, **kwargs):
         # TODO: May need to be changed depending on how username comes in from ODK?
         self.username = self.email
         super(User, self).save(*args, **kwargs)
+        if self.original_password != self.password:
+            UserPasswordHistory.remember_password(self)
+
+
+class UserPasswordHistory(models.Model):
+    username = models.ForeignKey(User, on_delete=models.CASCADE)
+    old_password = models.CharField(max_length=128)
+    password_date = models.DateTimeField()
```

Comment:
You can add `auto_now_add=True` to this field so you don't have to manually set the value in `remember_password`

Reply:
Thanks, that's good to know. I've changed this in the latest commit 8e0ab3d